.github/scripts/process_commit.py

```diff
@@ -70,10 +70,14 @@ def get_repo_labels() -> List[str]:
     return collected_labels
 
 def post_pytorch_comment(pr_number: int, merger: str) -> Any:
-    message = {'body' : f"Hey {merger}. \
-    You merged this PR, but no release notes category and topic labels were added. \
-    The list of valid release and topic labels is available \
-    https://github.com/pytorch/pytorch/labels?q=release+notes+or+topic"}
+    message = {'body' : f"""Hey {merger}. \
+    You merged this PR, but no it does not have both a 'release notes: ...' and 'topics: ...' label. \
+    Please add one of each to the PR. The 'release notes: ...' label should represent the part of \
+    PyTorch that this PR changes (fx, autograd, distributed, etc) and the 'topics: ...' label should \
+    represent the kind of PR it is (non-user visible, new feature, bug fix, perf improvement, etc). \
+    The list of valid labels can be found [here](https://github.com/pytorch/pytorch/labels?q=release+notes) \
```

Comment:
We should change query. I suggest something like :
https://github.com/pytorch/pytorch/labels?q=mandatorylabel
And add mandatorylabel to description of each required label

Reply:
This should fix the fact that https://github.com/pytorch/pytorch/labels?page=2&q=release+notes will result in some extra labels like "module: binaries" and cherry-picked

Reply:
From offline discussion, we agreed that the current query is not perfect but we don't really have a strictly better solution for now. Will keep the current for now and we can re-update in the future if we have a new idea.